experimental/telemetry/tracker.go

```diff
@@ -55,6 +62,7 @@ func NewTracker(
 	pluginVersion,
 	telemetryShortName string,
 	enableDiagnostics bool,
+	l logger.Logger,
```

Comment:
2/5 on backwards compatibility.

What's the current strategy for that? Should I be more conservative?

Reply:
My take (0/5) is that breaking backward compatibility is ok, as it will make (if someone bumps their version of the plugin API) the plugin not compile, so it's immediately identifiable. As long as we have it in the release notes as a breaking change, it should be fine to have the logger available in the tracker.

Reply:
great! I'm glad to know those boundaries